tests/sentry/api/test_api_owners.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase
+
+from sentry.api.api_owners import ApiOwner
+
+
+class APIOwnersTestCase(TestCase):
+    teams = set()
+
+    def setUp(self):
+        super().setUp()
+        code_owners_file = open(".github/CODEOWNERS")
+        lines = code_owners_file.readlines()
+        code_owners_file.close()
+        for line in lines:
+            if line.find("src/sentry/api/endpoints/") != -1:
```

Comment:
This leaves out endpoints defined outside of this folder - what if a team only owns endpoints outside of `/endpoints`?

Reply:
Good point. I take the endpoints out but still trust the teams to at leas use api/ folder.

Reply:
Sometimes we break code up by feature - for example - the `scim` folder exists at the same level as `api` and has it's own `endpoints` folder (see [here](https://github.com/getsentry/sentry/tree/master/src/sentry/scim)). I think it's fine for the test to miss these for now since they're not too common - but this might cause issues later

Reply:
yeah, this won't work. many teams put endpoints within their own folders.

Reply:
I can make this test to look in codeowners in general and see if this team owns any code :/ At least it would cover everything. I have another test coming specifically for APIs so should be fine.